### PR TITLE
Templates for standard mySociety footer on fixmystreet.org

### PR DESCRIPTION
**Pull request by zarino**
The templates to take advantage of https://github.com/mysociety/mysociety-docs-theme/pull/21

screen shot 2016-05-06 at 11 35 47

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <div class="col-sm-4">
+                <div class="mysoc-footer__legal">
+                    <p>mySociety Limited (05798215) is a project of UK Citizens Online Democracy (UKCOD) a registered charity in England and Wales, charity number 1076346, company number 03277032.</p>
+                </div>
+            </div>
+
+            <div class="col-sm-3">
+                <ul class="mysoc-footer__badges">
+                    <li role="presentation"><a href="https://github.com/mysociety/fixmystreet" class="mysoc-footer__badge mysoc-footer__badge--github">Github</a></li>
+                    <li role="presentation"><a href="https://www.twitter.com/mysociety" class="mysoc-footer__badge mysoc-footer__badge--twitter">Twitter</a></li>
```
Comment (dracos) [inline _includes/footer.html]: I wonder if fixmystreet is better twitter account, though that's about UK site, hmm.

File: _includes/footer.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+            <div class="col-sm-4">
+                <nav class="mysoc-footer__links">
+                    <ul>
+                        <li role="presentation"><a href="{{ site.baseurl }}install/">Install</a></li>
+                        <li role="presentation"><a href="{{ site.baseurl }}overview/">Documentation</a></li>
+                        <li role="presentation"><a href="https://github.com/mysociety/fixmystreet/blob/master/LICENSE.txt">Licence info</a></li>
+                    </ul>
+                    <ul>
+                        <li role="presentation"><a href="{{ site.baseurl }}community/">Mailing list</a></li>
```
Comment (dracos) [inline _includes/footer.html]: I think just Contact instead of these two, given they go to the same place.

File: _includes/footer.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                    <p>Open source report-mapping software that can be deployed anywhere in the world. Most commonly used for reporting street issues to councils, but flexible enough to fit any project that matches geographical points to email addresses.</p>
+                </div>
+            </div>
+
+            <div class="col-sm-4">
+                <nav class="mysoc-footer__links">
+                    <ul>
+                        <li role="presentation"><a href="{{ site.baseurl }}install/">Install</a></li>
+                        <li role="presentation"><a href="{{ site.baseurl }}overview/">Documentation</a></li>
+                        <li role="presentation"><a href="https://github.com/mysociety/fixmystreet/blob/master/LICENSE.txt">Licence info</a></li>
```
Comment (dracos) [inline _includes/footer.html]: Would drop this link.

File: _includes/footer.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                    <p class="mysoc-footer__org">
+                        Built by
+                        <a href="https://www.mysociety.org" class="mysoc-footer__org__logo mysoc-footer__org__logo--mysociety">mySociety</a>
+                    </p>
+                </div>
+            </div>
+
+            <div class="col-sm-4">
+                <div class="mysoc-footer__legal">
+                    <p>mySociety Limited (05798215) is a project of UK Citizens Online Democracy (UKCOD) a registered charity in England and Wales, charity number 1076346, company number 03277032.</p>
```
Comment (dracos) [inline _includes/footer.html]: Drop this.

File: _includes/footer.html (hunk 1 above)
Comment (MyfanwyNixon) [inline _includes/footer.html]: Personally I think for this audience, better to point them to @mysociety.

**Comment by dracos** [comment]
This is missing the theme submodule update, just doing that now.

**Comment by dracos** [comment]
Note, try to have at least sass 3.4.15 as that changed its colour rounding and otherwise you end up with some colours varying by one point.

**Pull request merged**